### DISCUSSION TITLE
ComboStrap and ckgeditor editor fail when using a code block.

### DISCUSSION THREAD

**Discussion by seraphire** [combostrap/combo#32] [Q&A]
I like using the ckgeditor because I can copy and paste HTML into dokuwiki and it reformats the page content into the correct markdown.

This works very well for most elements, but it is failing with code blocks.

If I enter a code block as:

```
<code xml>
<node>
  <subnode />
</node>
</code>
```

it will flatten down in a code line format and lose all internal formatting. 

Additionally, it fails with <H1>, <H2> type tags and converts to literal equal signs in the rendered code.  Basic formatting, such as bold, italic, links and associated elements, even tables, appear to convert correctly.

Is anyone aware of this problem and what I can do to address it?

**Comment by gerardnico** [comment]
Ckgeditor takes the clipboard and parse it as html before transforming it in wiki markup.

It seems that it doesn't understand the html that you paste.

You should open a ticket to their repository.

Combostrap does not come with an editor. We show the installed editor. If it works with the default dokuwiki template, we may have a bug template but it's working with other plugin editor such as autotab.
